cmake/FindhipSYCL.cmake

```diff
@@ -0,0 +1,37 @@
+find_program(SYCLCC_EXECUTABLE syclcc-clang
+    PATH_SUFFIXES bin)
+
+if(SYCLCC_EXECUTABLE MATCHES "-NOTFOUND")
+  message(SEND_ERROR "Could not find hipSYCL syclcc-clang compiler")
+endif()
+
+set(CMAKE_C_COMPILER    ${SYCLCC_EXECUTABLE})
+set(CMAKE_CXX_COMPILER  ${SYCLCC_EXECUTABLE})
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Is there any particular reason C++14 is required? SYCL 1.2.1 is C++11.

Reply:
Yes, hipSYCL requires C++14.